queries.sql

```diff
@@ -86,4 +86,64 @@ SELECT owners.full_name, COUNT(animals.id) AS num_animals_owned
 FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id
-ORDER BY num_animals_owned DESC;
\ No newline at end of file
+ORDER BY num_animals_owned DESC;
+
+SELECT animals.name 
+FROM animals 
+JOIN visits ON animals.id = visits.animal_id 
+JOIN vets ON vets.id = visits.vet_id 
+WHERE vets.name = 'William Tatcher' 
+ORDER BY visits.visit_date DESC 
+LIMIT 1;
+
+SELECT COUNT(DISTINCT animal_id) FROM visits 
+WHERE vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
```

Comment:
- [ Required ] I think this query will not give you the _Different animals Stephanie Mendez saw_, to do that you need to create joins between the `animals`, `visits`, and `vets` tables. You can do this by refactoring your query as shown below.

```ruby
  SELECT DISTINCT animals.name
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez';
```